lib/attribute.js

```diff
@@ -455,7 +456,10 @@ validators.format = function validateFormat (instance, schema, options, ctx) {
   if (typeof instance === 'undefined') {
```

Comment:
Why not change this to `if (instance === undefined || instance === null) {`, and then just do `instance.toString()` below?